pkg/common/upgrade/managed.go

```diff
@@ -223,6 +238,49 @@ func createPodDisruptionBudgetWorkloads(h *helper.H) error {
 	return nil
 }
 
+// createNodeDrainWorkload creates test workload to test Node Drain functionality
+// which the managed-upgrade-operator is expected to be able to handle
+func createNodeDrainWorkload(h *helper.H) error {
```

Comment:
To reduce code duplication between this and `createPodDisruptionBudgetWorkloads` which are very similar, it's probably nicer to refactor them both into something generic (eg. `createManagedUpgradeWorkload`) which can accept the workload that need creating as an parameter.

Reply:
@mrbarge - Agreed.. That's a nice suggestion.. Will work on it tomorrow mostly since today will be occupied in hackday ahead.. Will update soon on it.. Thanks

Reply:
Added changes.. I think it would be better to separate out these workload constants to https://github.com/openshift/osde2e/blob/main/pkg/common/config/config.go which can be the sole place to add/remove workload vars eventually along with other vars and we can still have the same functionality.. For now, I kept it simple.. Let me know if you prefer having it any other way..